earthfile2llb/converter.go

```diff
@@ -1143,6 +1181,23 @@ func (c *Converter) Import(ctx context.Context, importStr, as string, isGlobal,
 	return c.varCollection.Imports().Add(importStr, as, isGlobal, currentlyPrivileged, allowPrivilegedFlag)
 }
 
+// Cache handles a `CACHE` command in a Target.
+// It appends run options to the Converter which will mount a cache volume in each successive `RUN` command,
+// and configures the `Converter` to persist the cache in the image at the end of the target.
+func (c *Converter) Cache(ctx context.Context, path string) error {
+	err := c.checkAllowed(cacheCmd)
+	if err != nil {
+		return err
+	}
```

Comment:
I think that there needs to be a call to `c.nonSaveCommand()` as well here.

```suggestion
	}
	c.nonSaveCommand()
```

Reply:
Added this